### DISCUSSION TITLE
The locale 'es' breaks the url of my site

### DISCUSSION THREAD

**Discussion by KempisGV** [i18next/next-i18next#1797] [Q&A]
Hello 😁, until now I was working with the locales 'en' and 'de', today I added 'es'. Adding this new locale is giving me problems in the url of the site, to all sites, no matter if it is 'de' or 'en' it is adding the /es like this localhost:3000/es, this is only happening when adding 'es', I tried adding 'mx' and it works correctly, however, I need to use 'es', does anyone know why this is happening?

My next-i18next.config.js file looks like this:

`module.exports = {
  i18n: {
    defaultLocale: 'en',
    locales: ['en', 'de', 'es'],
  },
};`

**Comment by adrai** [comment]
Please provide a reproducible example, and if you find an issue, create an issue (not a discussion).
